src/about-menu/open-about-menu.ts

```diff
@@ -0,0 +1,51 @@
+import { BrowserWindow, app, shell } from 'electron';
+import path from 'path';
+
+let aboutWindow: BrowserWindow | null = null;
+
+export function openAboutWindow() {
+	const aboutPath = path.join( __dirname, 'menu', 'about-menu.html' );
+
+	if ( aboutWindow ) {
+		aboutWindow.focus();
+		return;
+	}
+
+	aboutWindow = new BrowserWindow( {
+		width: 284,
+		height: 284,
+		resizable: false,
+		minimizable: false,
+		maximizable: false,
+		modal: true,
+		webPreferences: {
+			contextIsolation: true,
+		},
+	} );
+
+	aboutWindow.loadFile( aboutPath );
+
+	// Open external links in the default browser
+	aboutWindow.webContents.setWindowOpenHandler( ( { url } ) => {
+		shell.openExternal( url );
+		return { action: 'deny' };
+	} );
+
+	// Read package.json and pass version to about window
+	const packageJson = app.getVersion();
+
+	aboutWindow.webContents.on( 'dom-ready', () => {
+		if ( aboutWindow ) {
+			// Inject version into the about window's HTML
+			aboutWindow.webContents
+				.executeJavaScript( `document.getElementById('version').innerText = '${ packageJson }'` )
+				.catch( ( err ) => {
+					console.error( 'Error executing JavaScript:', err );
+				} );
```

Comment:
I am injecting the version number here into the HTML of the page. I tried a couple of things but this seemed to be the most straightforward way.